### PR TITLE
github.py: Fix the link to github notifications to include the notifications scope automatically for the user

### PR DESCRIPTION
**Pull request by su8**

### CODE REVIEW

File: py3status/modules/github.py [hunk 1]
```diff
@@ -4,7 +4,8 @@
 
 To check notifications a Github `username` and `personal access token` are
 required.  You can create a personal access token at
-https://github.com/settings/tokens The only `scope` needed is `notifications`,
+https://github.com/settings/tokens/new?scopes=notifications&description=py3status-query-github
+The only `scope` needed is `notifications` is selected automatically for you,
```
Comment (lasers) [inline py3status/modules/github.py]: Greetings, `First-time contributor`! :wave: 

Can you rename the description to just `py3status`?

And we don't work with C/C++ here. Maybe you wanted to try this at `conky`? :blush:

Comment (su8) [reply py3status/modules/github.py]: My bad, too much C/C++ in my head :rofl:

**Comment by lasers** [comment]
@su8 Thank you for this contribution.

**Comment by su8** [comment]
Thanks for merging it :heart_eyes:

**Pull request merged**